src/vivarium_gates_child_iv_iron/tools/make_artifacts.py

```diff
@@ -97,7 +97,7 @@ def build_artifacts(
     check_for_existing(Path(output_dir), location, append, replace_keys)
 
     if location in metadata.LOCATIONS:
-            build_single(location, output_dir, replace_keys)
+        build_single(location, output_dir, replace_keys)
```

Comment:
Is this an issue in the research template? If so, can you add it to your open research template PR